rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -196,8 +196,17 @@ def _precompiled_apple_resource_bundle_impl(ctx):
         outputs = [output_bundle_dir],
     )
 
+    # See https://github.com/bazel-ios/rules_ios/pull/747 for context
+    xccurrentversions = [
+        (None, None, depset([f]))
```

Comment:
Is there only ever going to be a single xccurrentversion? I'm not too familiar with this part of CoreData but just trying to think through a more efficient depset creation if necessary.

Reply:
AFAIK yes that file is unique under one `.xcdatamodeld` directory, I couldn't find any docs confirming that though, just based on my experience. It would be hard to know what file determines the current version if there was more than one I think?

Reply:
The "plural" pattern here is just to be consistent with the type list, but I could make it be a single file indeed. Suggestions welcomed!

Reply:
I can also `fail` if more than one is found and then access index `[0]` and call it `xccurrentversion` instead of `xccurrentversions`. 

I honestly don't have a preference here, let me know if I should treat this as a nit or not. Otherwise will land this soon!

Reply:
Nah, all good on my side. I was just thinking passing the list to a single depset instead of N depset but I might be misunderstanding the datamodels input.

Reply:
Landing, happy to nit in a follow up!